### PR TITLE
sage.rb: Switch mirror; update to v7.2; fix URLs

### PR DESCRIPTION
**Pull request by claui**
#### Editing an existing cask
- [x] Commit message includes cask’s name (and new version, if applicable).
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
#### Details

For details, see the commit message.

### CODE REVIEW

File: Casks/sage.rb [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
-    url "http://mirrors.mit.edu/sage/osx/intel/sage-#{version}-x86_64-Darwin-OSX_10.10_x86_64-app.dmg"
+    url "http://mirrors.mit.edu/sage/osx/intel/old/sage-#{version}-x86_64-Darwin-OSX_10.10_x86_64-app.dmg"
   else
-    version '7.1'
-    sha256 'ce900a39dd1a4c33cec40a3c2fde81cf4a4446a7d8b3985041b501b9d88f8471'
-    # go-parts.com/sage was verified as official when first introduced to the cask
-    url "http://mirrors-usa.go-parts.com/sage/sagemath/osx/intel/sage-#{version}-OSX_10.11.3-x86_64.app.dmg"
+    version '7.2'
+    sha256 '4ede3cac113778e120d233baf702426ffad10905d43afe239c1ea8fd1068697e'
+    # mit.edu/sage was verified as official since the 7.2 version bump
```
Comment (vitorgalvao) [inline Casks/sage.rb]: Message needs to remain as it was. It’s standard.

**Comment by claui** [comment]
Thanks @vitorgalvao! Fixed.

File: Casks/sage.rb [hunk 2]
```diff
@@ -3,17 +3,17 @@
...
     sha256 '03112bf747cf807f308d518f34c1982ca3c9599e65bf64a6782efc78136198a4'
     # mit.edu/sage was verified as official when first introduced to the cask
-    url "http://mirrors.mit.edu/sage/osx/intel/sage-#{version}-x86_64-Darwin-OSX_10.10_x86_64-app.dmg"
+    url "http://mirrors.mit.edu/sage/osx/intel/old/sage-#{version}-x86_64-Darwin-OSX_10.10_x86_64-app.dmg"
   else
-    version '7.1'
-    sha256 'ce900a39dd1a4c33cec40a3c2fde81cf4a4446a7d8b3985041b501b9d88f8471'
+    version '7.2'
+    sha256 '4ede3cac113778e120d233baf702426ffad10905d43afe239c1ea8fd1068697e'
     # go-parts.com/sage was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/sage.rb]: Apologies for the confusion. I meant it should be `# mit.edu/sage was verified as official when first introduced to the cask`. Changing the first part was correct, only the rest of it should have remained.

**Comment by claui** [comment]
Here we go!

**Comment by vitorgalvao** [comment]
Thank you.

**Pull request merged**